### PR TITLE
Added blogpost for each project

### PR DESCRIPTION
**Pull request by Achint08**
Added Link to Blog posts and link to libraries for each project.

### CODE REVIEW

**Comment by niccokunzmann** [comment]
@Achint08 thanks for listing the blog posts. However, they need to be listed in the README.md file. The index.html file gets updated from the README.md file with the gh-pages editor in the settings. I see I should clarify this!

**Comment by niccokunzmann** [comment]
Added it: https://github.com/fossasia/knitting.fossasia.org/blob/gh-pages/Contributing.md
Thanks for pointing it out.

**Comment by Achint08** [comment]
I have removed links from index.html and modified README.md. Please check if its fine now.

File: README.md [hunk 1]
```diff
@@ -60,7 +63,10 @@ Contribute:
 ### knittingpattern [![View on Github][github-logo]][knittingpattern-repo]
 
 The `knittingpattern` is a Python library for an exchange format for knit work.
-It allows specifying how to knit a particular piece of work.
+It allows specifying how to knit a particular piece of work. You can
+
+- read the [blog post about this project][http://blog.fossasia.org/knitting-pattern-conversion/].
```
Comment (niccokunzmann) [inline README.md]: Can you add the blog links to the sections below?

```
- read the [blog post about this project][knittingpattern-blog-conversion].
```

and 

```
[knittingpattern-blog-conversion]: http://blog.fossasia.org/knitting-pattern-conversion/
```

The motivation is that links a) may change and b) clutter the page when read in markdown c) may be used several times.

File: README.md [hunk 2]
```diff
@@ -60,7 +63,10 @@ Contribute:
...
+- check out the [libraries][https://pypi.python.org/pypi/knittingpattern].
```
Comment (niccokunzmann) [inline README.md]: I.e. this link is already there:

```
[knittingpattern-pypi]: https://pypi.python.org/pypi/knittingpattern
```

File: README.md [hunk 3]
```diff
@@ -82,7 +88,11 @@ Contribute:
 
 The `AYABInterface` is used by the [Knit Editor][kniteditor] to
 convert [knittingpatterns][knittingpattern] into machine instructions.
-It can be used standalone without these other projects.
+It can be used standalone without these other projects.You can
```
Comment (niccokunzmann) [inline README.md]: space missing.

File: README.md [hunk 4]
```diff
@@ -147,7 +162,11 @@ Knitlib is based on projects like [AYAB][ayab], PDD, and KnitterStream to contro
 Knitlib features a plugin system for knitting machines and implements an API to control machines' operation,
 knitting jobs and knitting patterns. The software is based on Python.
 There also is a Web API.
-Among the primary tasks is to develop plugins based on this solution to add support for more machines.
+Among the primary tasks is to develop plugins based on this solution to add support for more machines.You can
```
Comment (niccokunzmann) [inline README.md]: Space missing

**Comment by niccokunzmann** [comment]
@Achint08 I will merge it now. There are conflicts that I caused, so I will do it locally.

**Comment by Achint08** [comment]
Added spaces and tags for links and removed extra tags.

**Comment by Achint08** [comment]
It was my first contribution. Thanks for your guidance. :)

**Comment by niccokunzmann** [comment]
@Achint08 thanks :) Your contribution is welcome. I will close this now.

**Pull request closed without merging**